targets/ath79-generic

```diff
@@ -119,13 +119,13 @@ device('d-link-dir-505', 'dlink_dir-505', {
 	},
 })
 
-device('d-link-dir825b1', 'dlink_dir-825-b1', {
-	factory = false,
-	class = 'tiny', -- Only 6M of usable Firmware space
-	manifest_aliases = {
-		'd-link-dir-825-rev-b1', -- Upgrade from OpenWrt 19.07
-	},
-})
+-- device('d-link-dir825b1', 'dlink_dir-825-b1', {
```

Comment:
This only affects factory-installation. Usable space is 7808k when upgrading from older versions.

Two ideas:

1. Backport https://github.com/openwrt/openwrt/commit/aca8bb5cc332f0ffdf4249e76b0a56716f98bef0 to the upcoming Gluon release to provide upgrade-path to newer image
2. Disable factory image generation in Gluon by either patching OpenWrt or disabling factory image build for this device exclusively.

Reply:
`factory` is already set to false. Or is that not "strong enough"?

I disabled it because the build failed otherwise. But maybe i've misinterpreted the build error.

Reply:
https://github.com/openwrt/openwrt/commit/aca8bb5cc332f0ffdf4249e76b0a56716f98bef0 was already backported to openwrt-22.03, so we should already have the expansion: https://github.com/openwrt/openwrt/commit/be2da47ced2fd0842922fdbd7a428a19138f6fc2